pygmt/src/ternary.py

```diff
@@ -11,6 +11,7 @@
     C="cmap",
     G="fill",
     JX="width",
+    L="label",
```

Comment:
`label` is a bad name, because it's already an alias for `-l` option.

I'm wondering if we should use a single parameter `labels=["a", "b", "c"]` or three parameters `alabel="a", blabel="b", clabel="c"`.

Reply:
I would like the idea of having three parameters to directly adjust the desired vertices without considering a potential order when using `labels=["a", "b", "c"]`.

Reply:
I also feel that three separate parameters are more Pythonic, although it needs more coding efforts.

Reply:
Great, I added 3 separate parameters. Do we need a unit test for this change although it's a quite simple implementation @GenericMappingTools/pygmt-maintainers ?

Reply:
Yes, I think a unit test or two would be good. Could you write a test for when all of alabel/blabel/clabel (3/3) are used, and one when only one of them (1/3) is used?

Reply:
Done!